src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+server.port=3333
```

Comment:
why do you want to set it? If you need for local testing you can set it for `local` profile only

Reply:
Fixed